ui/imports/shared/views/chat/ProfileContextMenu.qml

```diff
@@ -1,297 +1,215 @@
 import QtQuick 2.15
-
-import StatusQ.Popups 0.1
-import StatusQ.Components 0.1
-import StatusQ.Core.Utils 0.1 as StatusQUtils
+import QtQuick.Controls 2.15
 
 import AppLayouts.Chat.stores 1.0 as ChatStores
-
-import utils 1.0
-import shared 1.0
-import shared.panels 1.0
-import shared.popups 1.0
 import shared.status 1.0
-import shared.controls.chat 1.0
-import shared.controls.chat.menuItems 1.0
+import utils 1.0
 
+import "./ProfileContextMenu" as PCM
 
-StatusMenu {
+Item {
     id: root
 
     property ChatStores.RootStore store
```

Comment:
this is not in use actually, it should be removed from here and the sub components